sites/skeleton.dev/src/lib/layouts/DocsShell/partials/Header.svelte

```diff
@@ -10,7 +10,8 @@
 	const cChip = 'chip variant-soft hover:variant-filled';
 
 	// Local
-	const githubSourcePath = 'https://github.com/skeletonlabs/skeleton/tree/master/packages/skeleton/src'; // FIXME: hardcoded path
+	const githubSourcePath = 'https://github.com/skeletonlabs/skeleton/tree/dev/packages/skeleton/src';
+	const githubDocSourcePath = 'https://github.com/skeletonlabs/skeleton/blob/dev/sites/skeleton.dev/src';
```

Comment:
Should these be pointing at `dev` or `master`? Previously, it was pointed to `master`.

Reply:
Corrected, thanks!